Dockerfile

```
@@ -1,7 +1,7 @@
-FROM golang:1.14-buster AS builder
+FROM golang:1.15-buster AS builder
```

cant do this yet. Need to wait for https://github.com/dvyukov/go-fuzz/issues/294 to get fixed. 

```suggestion
FROM golang:1.14-buster AS builder
```